src/main/java/net/sf/jabref/importer/EntryFromPDFCreator.java

```diff
@@ -97,7 +97,7 @@ private void addEntryDataFromPDDocumentInformation(File pdfFile, BibEntry entry)
                         // default time stamp follows ISO-8601. Reason: https://xkcd.com/1179/
                         String date = new SimpleDateFormat("yyyy-MM-dd")
                                 .format(creationDate.getTime());
-                        appendToField(entry, InternalBibtexFields.TIMESTAMP, date);
+                        appendToField(entry, Globals.prefs.get(JabRefPreferences.TIME_STAMP_FIELD), date);
```

Comment:
Ok, I see. We might consider to drop this configuration option at some point in time. I don't really see why you might want to configure this.